Applications/HWC/start.sh

```
@@ -0,0 +1,32 @@
+
+#!/bin/bash
+set -e
+
+# Optionally start common services
+start-common.sh
+
+# Set AMRIT_HOME to the parent directory if not already set
+AMRIT_HOME="$(dirname $(dirname $(realpath $0)))"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)